src/ensembl/src/content/app/entity-viewer/components/views/gene-view/GeneView.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
perhaps move the `views` directory to the same level as `components`?

Or do we want a more substantial reshuffling, in a separate PR, to achieve something like:

```
--entity-viewer
    |
    |--views
        |
        |--gene-view
            |
            |--components
            ...
```

Reply:
```
--entity-viewer
    |
    |--gene-view
        |
        |--components
```

I vote for this one.

Reply:
This looks better and you both agree 😄 So I'll change the directory structure to this.

Reply:
```
--entity-viewer/
    |
    |--entity-viewer-app-bar/
    |--entity-viewer-sidebar/
    |--entity-viewer-sidebar-tabs/
    |--gene-view/
         |
         |--feature-length-axis/
    |--state/
    |--EntityViewer.tsx
```

I suggest we remove the existing `components` folder so that we can include state files inside the view folders.

Reply:
![image](https://user-images.githubusercontent.com/6834224/73550050-cbbad600-443b-11ea-9a7f-7deb8bb326e3.png)

Perhaps there can be a `components` (or maybe `shared/components`) folder as the first child of `entity-viewer` where these things can go?

Reply:
Should they be shared though? These components are going to always be in EV, despite the view changing. On the other hand, if we put them in `components`, then where should we put the views, which happen to be components themselves?

Reply:
After the discussion I had with @azangru, we decided to use the following directory structure:

```
--entity-viewer/
    |
    |--shared/
         |--components/
              |--app-bar/
              |--view-in/
         |--state
         |--helpers
    |--gene-view
         |--components/
         |--state/
         |--helpers
    |--transcript-view
         |--components/
         |--state/
         |--helpers/
```